MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistantly when using Mono or Microsoft compilers
+        /// </summary>
+        public class Random
```

Comment:
Before we merge we should probably have one more discussion on naming here.

`Microsoft.Xna.Framework.Random` isn't an option because it would collide with `System.Random` all the time.

`Microsoft.Xna.Framework.GameRandom` is a little long and the prefix doesn't feel all that descriptive, but allows you to use `GameRandom` everywhere easily.  Maybe there is a better `FooRandom` name to be found?

`Microsoft.Xna.Framework.MathHelpers.Random` lets you use `MathHelpers.Random` which is maybe more descriptive... random is math not game.  Howerver calling any static method on it is ugly...  `var seed = MathHelpers.Random.GenerateSeed()`.

Anyway... felt like it was worth one more discussion.